sympy/core/tests/test_mod.py

```diff
@@ -0,0 +1,11 @@
+from sympy.utilities.pytest import raises
+from  sympy.core.numbers import nan
+from sympy.core.numbers import oo, zoo
+from sympy.core.mod import Mod
+
+def test_mod():
```

Comment:
The tests for `Mod` should be added here:
https://github.com/sympy/sympy/blob/master/sympy/core/tests/test_numbers.py#L49